common/scpi_cmds.c

```diff
@@ -61,6 +64,70 @@ static uint32_t scpi_cmd_get_scp_cap_tx_payload[] = {
 	0,
 };
 
+/*
+ * Handler/payload data for SCPI_CMD_SET_CSS_PWR: Set CSS power state.
+ *
+ * The power state provided by PSCI is already coordinated. Simply turn things
+ * on from highest to lowest power level, then turn things off from lowest to
+ * highest power level. This ensures no power domain is turned on before its
+ * parent, and no power domain is turned off before any of its children.
+ */
+uint32_t
+scpi_cmd_set_css_pwr_handler(uint32_t *rx_payload, uint16_t rx_size __unused,
+                             uint32_t *tx_payload __unused,
+                             uint16_t *tx_size __unused)
+{
+	uint32_t descriptor    = rx_payload[0];
```

Comment:
`scpi_cmd_set_css_pwr_handler` seems to assume only one descriptor while `scpi_cmd_get_css_pwr_handler` handles multiple. Is there a reason for this discrepancy?

Reply:
Because that's what the specification says? When you want to turn a CPU on, you have to tell it which one to turn on. But when you want to get the state of the CPUs, there's no parameter. It just gives you all of them